addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1260,6 +1262,7 @@ options.registry.OptionsTab = options.Class.extend({
         const widget = this._requestUserValueWidgets(params.imagepicker)[0];
         widget.enable();
     },
+
```

Comment:
At least in legacy code (maybe it will be like that in OWL classes), don't add those extra blank lines between methods.